inlong-common/src/main/java/org/apache/inlong/common/pojo/agent/TaskSnapshotRequest.java

```diff
@@ -23,31 +23,29 @@
 import java.util.Date;
 import java.util.List;
 
-/**
- * Request of the agent task snapshot
- */
 @Data
 public class TaskSnapshotRequest {
 
     /**
-     * The ip of agent
```

Comment:
Why use lower-case instead of upper-case?

Reply:
huve update the codes